### DISCUSSION TITLE
Hide the "on" "off"

### DISCUSSION THREAD

**Discussion by sierralx** [cgiesche/streamdeck-homeassistant#311] [Help and Support]
Hello. Is there a way to hide the text "on" and "off" below the icon? I couldn't find any option to do so in the latest update.

**Comment by cgiesche** [comment]
Hi sierralx,

I removed the default label for all on/off states. This configuration is updated automatically by the plugin from time to time (may take up to 7 days due to caching).

If you need an instant fix, you can apply this workaround: https://github.com/cgiesche/streamdeck-homeassistant/discussions/308.
